index.js

```diff
@@ -356,6 +356,22 @@
     ($.Undefined  = type0('Undefined'))
   ];
 
+  //  Enum :: [Any] -> Type
+  $.Enum = function(constants) {
+    var reprs = map(constants, show);
+    return {
+      type: 'sanctuary-def/Enum',
```

Comment:
Let's change this value to `'ENUM'`, for consistency with `'NULLARY'`, `'UNARY'`, `'BINARY'`, `'RECORD'`, etc.